src/strategies/cargo.test.ts

```diff
@@ -56,26 +50,23 @@ version = "0.1.1"
       'packages/cargo-vs',
     );
 
-    container.register('cwd', {
-      useValue: 'packages/cargo-vs',
-    });
-    container.register(Git, {
-      useClass: Git,
-    });
-    container.register(VersionStrategy, {
-      useClass: CargoVersionStrategy,
-    });
+    container.bind('cwd').toConstantValue('packages/cargo-vs');
+    container.bind<Git>(Git).to(Git);
+    // container.bind(Git, {
+    //   useClass: Git,
+    // });
+    container.bind<VersionStrategy>(VersionStrategy).to(CargoVersionStrategy);
 
     await t.step('Test GetCurrentVersion', async () => {
-      const vsInstance = container.resolve(VersionStrategy);
-      const currentVersion = await vsInstance.getCurrentVersion();
-      assertEquals(currentVersion, '0.1.1');
+      // const vsInstance = container.resolve<VersionStrategy>(VersionStrategy);
```

Comment:
have some testing to do about this still -- forgot I had commented this out while working on something.